docs/chainlink-keepers/cron-contracts.md

```diff
@@ -0,0 +1,176 @@
+```
+
+```json
+<your target address>, <your function specification>, <your encodedCronSpec>
+```
+Replace the values and paste the snippet above into the `createCronJobFromEncodedSpec` function in Remix and execute the function. Once executed, you can call the `getActiveCronJobIDs` to see if your job has been registered. You can execute the `getCronJob` function to view the details of your job played back. You can add multiple jobs on this contract in this fashion, and also use the other in your Job Scheduler to delete jobs or pause and unpause the Scheduler. However, you must register your Job Scheduler with the Chainlink Keepers network for it to be monitored and executed.
+
+
+## Setting up Chainlink Keepers
+
+Because the deployed contract is already [Keeper-compatible](../compatible-contracts), you can immediately register it with Keepers following these [steps](../register-upkeep). Be sure to provide the address of your Job Scheduler contract in **Upkeep address**. For gas limits, you should specify the upper limit of gas your target function will use.
```

Comment:
```suggestion
Because the deployed contract is already [Keeper-compatible](../compatible-contracts), you can immediately [register the contract for upkeep(/docs/chainlink-keepers/register-upkeep/). Provide the address of your Job Scheduler contract in the **Upkeep address** field. For gas limits, specify the upper limit of gas that your target function will use.
```